app/src/main/java/com/nononsenseapps/feeder/model/RssLocalSync.kt

```diff
@@ -49,6 +52,7 @@ class RssLocalSync(override val di: DI) : DIAware {
     private val feedParser: FeedParser by instance()
     private val okHttpClient: OkHttpClient by instance()
     private val filePathProvider: FilePathProvider by instance()
+    private val application: FeederApplication by instance()
```

Comment:
This caused a crash in my tests.
```suggestion
    private val application: Application by instance()
```

Reply:
Interesting. Will fix that.

Reply:
Fixed.